ucc/compile.py

```diff
@@ -56,12 +165,17 @@ def compile(
 
     # Translate to Qiskit Circuit object
     qiskit_circuit = translate(circuit, "qiskit")
+
+    # Utilize approximate quantum compilation via MPS encoding
```

Comment:
As an alternative implementation, would it make more sense for aqc to be a custom pass? Given the user has to opt-in, that seems to match other patterns and also the approach for another optional method in #412.